scripts/runSchoolApi.js

```diff
@@ -126,7 +126,7 @@ const createUser = async (baseUrl, apiKeys, schoolId, userType) => {
 			const { id } = await createSchool(schoolUrl, finalHeaders);
 			schoolId = id;
 		} else {
-			console.log(`Using existing school ID: ${schoolId}`);
+			console.log(`User created using existing school ID: ${schoolId}`);
```

Comment:
is this correct? What is meant here? 
"User created for existing school ID .."?

Reply:
yes each time user is created in the same school to display as a log message, as this log is part of createUser.